src/components/numeral-date/numeral-date.stories.mdx

```diff
@@ -30,164 +31,22 @@ import NumeralDate from "carbon-react/lib/components/numeral-date";
 
 ## Examples
 
-### Controlled
+### Default
 
 <Canvas>
-  <Story name="controlled">
-    {() => {
-      const [value, setValue] = useState({ dd: "", mm: "", yyyy: "" });
-      return (
-        <NumeralDate
-          value={value}
-          onChange={(e) => setValue(e.target.value)}
-          label="Default"
-        />
-      );
-    }}
-  </Story>
+  <Story name="default" story={stories.Default} />
```

Comment:
I'm happy to approve this change since I agree it makes sense, but wanted to highlight that chromatic will treat this as a new story due to the change in story name.

Reply:
I think that we have agreed that we'll be changing the stories during TS refactor